op-e2e/actions/proofs/helpers/kona.go

```diff
@@ -0,0 +1,81 @@
+package helpers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"math/big"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/utils"
+	"github.com/ethereum-optimism/optimism/op-challenger/game/fault/trace/vm"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-program/client/claim"
+	"github.com/stretchr/testify/require"
+)
+
+var konaHostPath, konaClientPath string
+
+func init() {
+	konaHostPath = os.Getenv("KONA_HOST_PATH")
+	konaClientPath = os.Getenv("KONA_CLIENT_PATH")
+}
+
+func IsKonaConfigured() bool {
+	return konaHostPath != "" && konaClientPath != ""
+}
+
+func RunKonaNative(
+	t helpers.Testing,
+	workDir string,
+	env *L2FaultProofEnv,
+	l1Rpc string,
+	l1BeaconRpc string,
+	l2Rpc string,
+	fixtureInputs FixtureInputs,
+) error {
+	// Write rollup config to tempdir.
+	rollupConfigPath := filepath.Join(workDir, "rollup.json")
+	ser, err := json.Marshal(env.sd.RollupCfg)
+	require.NoError(t, err)
+	require.NoError(t, os.WriteFile(rollupConfigPath, ser, fs.ModePerm))
+
+	// Run the fault proof program from the state transition from L2 block 0 -> 1.
+	vmCfg := vm.Config{
+		L1:               l1Rpc,
+		L1Beacon:         l1BeaconRpc,
+		L2:               l2Rpc,
+		Network:          "op-mainnet",
```

Comment:
Shouldn't we just omit the network given the rollup config is being specified? Seems very wrong to hard code op-mainnet here.